src/test/java/millom/sandbox/Utility/JsonUtility.java

```diff
@@ -0,0 +1,211 @@
+package millom.sandbox.Utility;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class JsonUtility {
+
+  public static final String SAMPLE_RESPONSE= """
```

Comment:
I see Java 17 became much more forgiving with it's multi-line strings :-)

Since this is intended for learning, can we try to put one or all JSONs to the resource and try to load them from there? That's a pretty common practical task. 

https://devtut.github.io/java/resources-on-classpath.html#loading-default-configuration

Reply:
Created 3 json files for 3 different json responses in resource folder for tests.
Created new method for accessing the json file from resource folder.

```
public static String getSampleResponse(String fileName) throws IOException {
    try (InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(fileName)) {
      JsonNode jsonNode = MAPPER.readValue(inputStream, JsonNode.class);
      return MAPPER.writeValueAsString(jsonNode);
    }
  }
```

And then trying to access the json through the above method. 
Example:
` Weather response = nasaMapper.deserializeWeather(JsonUtility.getSampleResponse("response.json"));`